### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by EugeneTeu**

### CODE REVIEW

**Comment by EugeneTeu** [comment]
My PR is getting quite big i would like to merge first

File: database/create_orders.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
+CREATE TABLE orders 
+(
+    order_id SERIAL PRIMARY KEY,
+    total_cost NUMERIC(10,2),
+    status text ,
```
Comment (yan-wl) [inline database/create_orders.sql:5]: Maybe this should be NOT NULL DEFAULT 'pending', or something similar

File: database/create_orders.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    promo_id SERIAL PRIMARY KEY,
+    order_id SERIAL,
+    discount DECIMAL(5,2), 
+    start_time DATE,
+    end_time DATE,
+    FOREIGN KEY (order_id) REFERENCES orders(order_id)
+);
+
+/*Order can only have one location*/
+CREATE TABLE locations
```
Comment (yan-wl) [inline database/create_orders.sql:21]: This table doesn't comply with the ER.
Think the From relationship set cannot be omitted.

File: database/create_orders.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+CREATE TABLE locations 
+(
+    order_id SERIAL PRIMARY KEY,
+    address text,
+    FOREIGN KEY (order_id) REFERENCES orders(order_id)
+);
+
+CREATE TABLE contain
+(
+    order_id SERIAL,
```
Comment (yan-wl) [inline database/create_orders.sql:30]: Not sure if SERIAL works here. Maybe change to INTEGER.

File: database/create_orders.sql [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+(
+    order_id SERIAL PRIMARY KEY,
+    address text,
+    FOREIGN KEY (order_id) REFERENCES orders(order_id)
+);
+
+CREATE TABLE contain
+(
+    order_id SERIAL, 
+    food_id SERIAL,
```
Comment (yan-wl) [inline database/create_orders.sql:31]: Similar to above.

File: database/create_orders.sql [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    quantity int,
```
Comment (yan-wl) [inline database/create_orders.sql:32]: Might want to make this NOT NULL and impose a positive quantity constraint.

File: database/create_restaurants.sql [hunk 6]
```diff
@@ -8,21 +8,15 @@ CREATE TABLE restaurants (
...
-  FOREIGN KEY (restaurant_id) REFERENCES restaurants (restaurant_id),
-  created_at TIMESTAMP DEFAULT current_timestamp,
-  modified_at TIMESTAMP DEFAULT current_timestamp
-);
-
 CREATE TABLE foods (
   food_id SERIAL PRIMARY KEY,
   category TEXT,
   food_name TEXT,
+  daily_limit int,
```
Comment (yan-wl) [inline database/create_restaurants.sql:15]: Might want to make this NOT NULL and impose positive constraint.

File: database/create_user.sql [hunk 7]
```diff
@@ -10,11 +10,18 @@ CREATE TABLE users
...
-    card text DEFAULT '0000',
-    num_orders int DEFAULT 0,
+    order_count int DEFAULT 0,
+    point int DEFAULT 0,
     FOREIGN KEY (customer_id) REFERENCES users (user_id)
 );
 
+CREATE TABLE cards  
+(
+    customer_id SERIAL,
```
Comment (yan-wl) [inline database/create_user.sql:20]: Not sure if SERIAL works

**Review by yan-wl — APPROVED** [review]
Only checked database schemas. Looks ok

**Pull request merged**